source/material_model/rheology/strain_dependent.cc

```diff
@@ -147,6 +147,25 @@ namespace aspect
                            "for background material and compositional fields, "
                            "for a total of N+1 values, where N is the number of compositional fields. "
                            "If only one value is given, then all use the same value.  Units: None.");
+
+        prm.declare_entry ("Strain healing mechanism", "no healing",
+                           Patterns::Selection("no healing|temperature dependent"),
+                           "Whether to apply strain healing to plastic yielding and viscosity terms, "
+                           "and if yes, which method to use. The following methods are available:"
+                           "\n\n"
+                           "\\item ``no healing'': No strain healing is applied. "
+                           "\n\n"
+                           "\\item ``temperature dependent'': Purely temperature dependent "
+                           "strain healing applied to plastic yielding and viscosity terms, "
+                           "as done in Fuchs and Becker, 2019");
```

Comment:
Can you add a line stating that the formulation closely follows that of the Frank Kemenetskii viscosity model, or something to that effect?